static/app/gettingStartedDocs/capacitor/capacitor.tsx

```diff
@@ -63,6 +63,12 @@ const platformOptions: Record<PlatformOptionKey, PlatformOption> = {
 type PlatformOptions = typeof platformOptions;
 type Params = DocsParams<PlatformOptions>;
 
+const getVerifyReplaySnippet = () => `
+setTimeout(() => {
+  throw new Error("Sentry Test Error");
+});
```

Comment:
I think button snippet would provide better first replay than `setTimeout`.

The same as we have for iOS and Android.

Reply:
I took this one from sentry-docs...
I guess we would need 3 different snippets. One for `Angular`, `React` and `Vue`?

Reply:
Looks like a wrong link. Yeah I see, it would make sense to add for Angular, React and Vue, but only if we already have some tab UI ready for these situations.

Otherwise I would just `throw new Error("Sentry Test Error");` and in desc tell users to add it to onClick.

Reply:
The link was added by our GitHub integration 😅 (as there is a project called `sentry-docs`)
There is some UI for it and in those 3 we will soon have docs that add a button. So I can re-use them 👍